packages/survey-creator-core/tests/question-converter.tests.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  Base,
+  SurveyModel,
+  QuestionRadiogroupModel,
+  QuestionCommentModel,
+  QuestionCheckboxModel,
+  QuestionTextModel
+} from "survey-core";
+import { QuestionConverter } from "../src/questionconverter";
+import { settings } from "../src/settings";
+
+test("get converted classes", () => {
+  var classes = QuestionConverter.getConvertToClasses("radiogroup");
+  expect(classes.length >= 3).toBeTruthy();
```

Comment:
JFYI. There is more suitable matcher function in Jest ― [toBeGreaterThanOrEqual ](https://jestjs.io/docs/expect#tobegreaterthanorequalnumber--bigint). Moreover [toBeTruthy](https://jestjs.io/docs/expect#tobetruthy) implemented with something like:
```javascript
return !!value;
```
That probably can bring some problems